android/src/main/java/com/apparence/camerawesome/CameraPicture.java

```diff
@@ -218,6 +223,10 @@ public void setAutoFocus(boolean autoFocus) {
         this.autoFocus = autoFocus && mCameraCharacteristics.hasAutoFocus();
     }
 
+    public void setRecordingAudioMode(boolean enableAudio) {
+        this.enableAudio = enableAudio;
+    }
```

Comment:
Since this isn't actually a audio mode setter but rather a audio enabled toggle, I'd rename it to something like this:
```suggestion
    public void setRecordAudioEnabled(boolean enableAudio) {
        this.enableAudio = enableAudio;
    }
```

Reply:
I totally agree but this is exactly the same as how it's called in iOS so I wanted to keep it consistent.

Reply:
maybe add this as a comment above the method for future reference?

Reply:
Updated the naming in the whole project.